port.sh

```
@@ -0,0 +1,39 @@
+  4000 # Talawa API
+  443  # Caddy HTTPS
+)
+
+echo "Checking port usage locally and in Docker containers..."
+
+for port in "${ports[@]}"; do
+  echo -e "\n==============================="
+  echo "  Checking port $port"
+
+  # Check if port is used locally
+  local_usage=$(sudo lsof -i :$port)
```

Shellcheck (0.10.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)